.gitignore

```
@@ -0,0 +1,44 @@
+/target/
+pom.xml.versionsBackup
+
+# Eclipse and Maven
+.classpath
+.project
+/.settings/org.eclipse.core.resources.prefs
+/.settings/org.eclipse.m2e.core.prefs
+/.settings/org.eclipse.jdt.apt.core.prefs
+# .settings : we need Eclipse settings for code formatter and clean-up rules
+target
+.cache
+dependency-reduced-pom.xml
+
+# Intellij
+.idea
+# Intellij recommends to share iml files, however, better don't share files which might be outdated
+*.iml
+
+
+# Others
+.DS_Store
+*.swp
+local
+Scripts
+.dbeaver*
+**/*.log
+.directory
+venv/
+
+~*
+*.lock
+*.bak
+*.orig
+*.old
+*.md.html
+/.apt_generated/
+/.apt_generated_tests/
+*.flattened-pom.xml
+/bin/
+
+username.txt
+password.txt
+accountname.txt
```

See below, proposing to replace the 3 files by a single one and also being easier to consume for a Java application or junit tests: 

```suggestion
test.properties
```

---

Done!